erpnext_documentation/www/docs/v13/user/manual/en/buying/buying-settings.md

```diff
@@ -5,7 +5,7 @@ Buying Settings is where you can define properties which will be applied in the
 You can find Buying Settings at:
 > Home > Buying > Settings > Buying Settings
 
-![Buying Settings]({{docs_base_url}}/v13/assets/img/buying/buying-settings.png)
+![Buying Settings]({{docs_base_url}}/v13/assets/img/buying/buying-settings-v13.png)
```

Comment:
Please highlight the 'Bill for Rejected Quantity For Purchase Invoice' with a rectangle in the screenshot.

You can check https://docs.erpnext.com/docs/v13/assets/img/buying/po-required.png

Also, should the label be 'Bill Rejected Quantity For Purchase Invoice' instead of 'Bill **for** Rejected Quantity For Purchase Invoice'?

In fact, 'Include Rejected Quantity in Purchase Invoice' would be better I guess. cc: @nabinhait

Reply:
Hey @asbasawaraj my bad. The label is actually "Bill for Rejected Quantity in Purchase Invoice" and not "Bill for Rejected Quantity For Purchase Invoice" and in my opinion it conveys the message clearer than the other.